client/cmd/up.go

```diff
@@ -13,121 +13,137 @@ import (
 	gstatus "google.golang.org/grpc/status"
 )
 
-var upCmd = &cobra.Command{
-	Use:   "up",
-	Short: "install, login and start Netbird client",
-	RunE: func(cmd *cobra.Command, args []string) error {
-		SetFlagsFromEnvVars()
+var (
+	daemonModeOff bool
+	upCmd         = &cobra.Command{
+		Use:   "up",
+		Short: "install, login and start Netbird client",
+		RunE:  upFunc,
+	}
+)
 
-		cmd.SetOut(cmd.OutOrStdout())
+func init() {
+	upCmd.PersistentFlags().BoolVar(&daemonModeOff, "daemon-off", false, "start service in foreground")
```

Comment:
It is more explicit if we call the flag `foreground-mode` as some users aren't aware of the service being called daemon. 

We can also use BoolVarP and add `F`  as shorthand for the flag.

Reply:
You are right!
Do you wish ```F``` in upper case?

Reply:
Yes, I think the upper case resembles better with the running mode.